topics/introduction/tutorials/galaxy-intro-101/tutorial.md

```diff
@@ -119,7 +117,7 @@ We are now ready to perform our analysis, but first we need to get some data int
 > 4. Let's rename our dataset to something more recognizable.
 >    - Click on the **pencil icon** to edit the dataset attributes.
 >    - In the next screen change the name of the dataset to `Exons`.
->    - Click the **Save** button at the bottom of the screen.
+>    - Click the **Save** button.
```

Comment:
What about using the snippet for renaming datasets here:

```
>    {% include snippets/rename_dataset.md %}
```

The content of the snippet:

```
>
>    > ### {% icon tip %} Tip: Rename a dataset
>    > * Click on the {% icon galaxy-pencil %} button displayed on each dataset in the history
>    > * Change the **Name** attribute
>    > * Press **Save**
>    {: .tip}
>
```

Reply:
For the Galaxy 101 that's not a tip, but a proper step, where the trainer explains why having recognizable names is important. Also the snippet would need to be extended to include the new name. Is it worth it?

Reply:
Good point for a proper step in 101